Lib/ufo2ft/kernFeatureWriter.py

```diff
@@ -9,27 +12,34 @@ class KernFeatureWriter(AbstractFeatureWriter):
     Uses the kerning rules contained in an RFont's kerning attribute, as well as
     glyph classes from parsed OTF text. Class-based rules are set based on the
     existing rules for their key glyphs.
+
+    Uses class attributes to match UFO glyph group names and feature syntax
+    glyph class names as kerning classes, which can be overridden.
     """
 
+    leftUfoGroupRe = r"^public\.kern1\.(.+)"
+    rightUfoGroupRe = r"^public\.kern2\.(.+)"
+    leftFeaClassRe = r"@_(.+)_L"
```

Comment:
No this syntax is just the only I'd seen, in Roboto. But that's why this is overrideable, let's change the default to `@MMK_L_(.+)`.